UCSD/campus-mobile
# delete offline alert on cards page to avoid double popup

## Summary
Issue [Card Management Online/Offline Status Does Not Change#1749](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1749) failed testing because of the double popups occurred. This is caused by the [offline mode](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1664) added earlier to enable offline alert while the cards page also enables offline alert.

This PR deletes the offline alert on cards. The earlier added offline mode is functional across the app.

## Changelog
[General] [Fix] - Fix the bug where the profile cards page has double offline alert


## Test Plan
1. go to profile/cards page
2. turn off the internet connection
3. An offline alert should pop up like below

<img width="260" alt="no_internet" src="https://user-images.githubusercontent.com/67001766/193445966-54743c94-5ffb-499a-91de-6e6d8b15bd84.png">
